### DISCUSSION TITLE
running a complete Cortex service in Kubernetes

### DISCUSSION THREAD

**Discussion by wgliang** [cortexproject/cortex#5175] [Q&A]
**Is your feature request related to a problem? Please describe.**
Is it possible to support running a complete Cortex service in Kubernetes with one click? For example, define the Yaml configuration of the service to simplify the deployment in kubernetes.

**Describe the solution you'd like**
Create a complete Cortex cluster with one or a few lines of commands.

**Describe alternatives you've considered**

**Additional context**

**Comment by alanprot** [comment]
you can use the helm chart for that..

You will still need to do some configs as which blocks storage you are using etc etc...

Btw you can use the CNCF slack channel for those questions..
